tests/registry_manager_test.py

```diff
@@ -0,0 +1,61 @@
+"""Tests for the kafkit.registry.manager module."""
+
+import os
+from pathlib import Path
+from typing import AsyncGenerator
+
+import pytest
+from aiohttp import ClientSession
+
+from kafkit.registry.aiohttp import RegistryApi
+from kafkit.registry.manager import RecordNameSchemaManager
+
+
+@pytest.fixture(scope="session")
+def schema_root() -> Path:
+    """Pytest fixture for the root path of schemas for testing the
+    RecordNameSchemaManager.
+    """
+    return Path(__file__).parent / "data" / "record_name_schemas"
+
+
+@pytest.mark.asyncio
+@pytest.fixture
+async def http_session() -> AsyncGenerator[ClientSession, None]:
+    """Pytest fixture for an aiohttp ClientSession."""
+    async with ClientSession() as session:
+        yield session
+
+
+@pytest.mark.docker
+@pytest.mark.skipif(
+    os.getenv("SCHEMA_REGISTRY_URL") is None,
+    reason="SCHEMA_REGISTRY_URL env var must be configured",
+)
+@pytest.mark.asyncio
```

Comment:
Is this needed any more? I seem to be able to write `async` tests without the decorator and `pytest` just copes.

Reply:
I think they are needed, I get:

```
  /Users/jsick/lsst/_square-events/kafkit/.tox/py37/lib/python3.7/site-packages/_pytest/python.py:157: PytestUnhandledCoroutineWarning: async def functions are not natively supported and have been skipped.
  You need to install a suitable plugin for your async framework, for example:
    - pytest-asyncio
    - pytest-trio
    - pytest-tornasync
    warnings.warn(PytestUnhandledCoroutineWarning(msg.format(pyfuncitem.nodeid)))
```

I guess you don't need the mark if you're using a fixture that's already using the asyncio mark.